feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/transaction/detail/transfer/TransactionDetailViewModel.kt

```diff
@@ -49,6 +58,15 @@ class TransactionDetailViewModel(
         .inBackground()
         .share()
 
+    val fiatFee = flowOf {
+        val fee = operation.fee ?: return@flowOf null
+        val currency = currencyInteractor.getSelectedCurrency()
+        val commissionAsset = chain.await().commissionAsset
+        val coinRate = coinPriceInteractor.getCoinPriceAtTime(commissionAsset.priceId!!, currency, operation.time.milliseconds.inWholeSeconds)
+        coinRate?.convertPlanks(commissionAsset, fee)
+            ?.formatAsCurrency(currency)
+    }.withSafeLoading()
```

Comment:
Dont forget to call `shareInBackground()`, otherwise this block will be computed on each subscription to flow